backend/lib/models/index.js

```diff
@@ -4,13 +4,15 @@ const { join } = require("path");
 const PATH_TO_SCHEMAS = join(__dirname, "schemas");
 
 module.exports = function registerModels(mongoConnection) {
+  require("./schemas/v2/post");
   readdirSync(PATH_TO_SCHEMAS).forEach((fileName) => {
     const filePath = join(PATH_TO_SCHEMAS, fileName);
 
     if (statSync(filePath).isDirectory()) return;
 
     const schema = require(filePath); // eslint-disable-line import/no-dynamic-require
-    const modelName = fileName.replace(".js", "");
+    var name = fileName.replace(".js", "");
```

Comment:
forgot a `var`
maybe we should add a linting rule if we don't want to allow this anyway
Also, maybe we should add a ticket to refactor this `registerModels` function, and eikther import the files all manually or update it so we have for every model, both a schema and a model file, what do you think is best @Naraujo13 ?

Reply:
Yep, probably a good idea to refactor.
I think that each file should export its schema and model and then `registerModels` can just import them all manually. I think manually would because the looping approach may be harder to control and may cause some undesired side-effects. Also, there are only a few models we should require, not something that lengthy, so I think the simpler approach would be best.

Reply:
Also, I forgot to add but I will change to `const` there but will leave the whole refactoring of the `registerModels` to another PR.

Reply:
yeah I would like to work on that, I can make a ticket for it
also we can add this https://eslint.org/docs/rules/no-var